packages/kbn-i18n/GUIDELINE.md

```diff
@@ -83,7 +83,7 @@ In case when `indicesLength` has value 1, the result string will be "`1 index`".
 
 The message ids chosen for message keys should always be descriptive of the string, and its role in the interface (button label, title, etc.). Think of them as long variable names. When you have to change a message id, adding a progressive number to the existing key should always be used as a last resort.
 
-- Message id should start with namespace (`kbn`, `common.ui` and etc.). 
+- Message id should start with namespace (`kbn`, `common.ui` and etc.) that identifies a plugin. 
```

Comment:
__nit:__ `a plugin` - not necessarily, we also have `common.ui` that isn't a plugin. Maybe something like "Message id should start with namespace that identifies a functional area of the app (`common.ui` or `common.server`) or a plugin (`kbn`, `vega` etc.)." or something along these lines.